tests fail on alpine linux

I am trying to write an APKBUILD file for nodeenv but the tests fail on all architectures apart from aarch64:
https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/11255

The error for x86_64 is about a url being wrong:
```
E           AssertionError: expected call not found.
E           Expected: urlopen('https://nodejs.org/download/release/index.json')
E           Actual: urlopen('https://unofficial-builds.nodejs.org/download/release/index.json')
/usr/lib/python3.8/site-packages/mock/mock.py:913: AssertionError
```

The other plattforms have a key_error eg:
`KeyError: 'armv8l'`

Apart from x86 which is:
`urllib.error.HTTPError: HTTP Error 404: Not Found`

--- a/tests/nodeenv_test.py
+++ b/tests/nodeenv_test.py
@@ -5,6 +5,7 @@ import os.path
 import pipes
 import subprocess
 import sys
+import sysconfig
 
 import mock
 import pytest
@@ -117,6 +118,11 @@ def test_mirror_option():
              'https://npm.some-mirror.com/download/release/index.json'),
             ('',
              'https://nodejs.org/download/release/index.json')]
+    sys_type = sysconfig.get_config_var('HOST_GNU_TYPE')
+    musl_type = ['x86_64-pc-linux-musl', 'x86_64-unknown-linux-musl']
+    # Check if running on musl system and delete last mirror if it is
+    if sys_type in musl_type:
+        urls.pop()
     with open(os.path.join(HERE, 'nodejs_index.json'), 'rb') as f:
         def rewind(_):
             f.seek(0)

